pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/io/buffer/UnsafeUtils.java

```diff
@@ -0,0 +1,295 @@
+package com.hedera.pbj.runtime.io.buffer;
+
+import java.lang.reflect.Field;
+import java.nio.Buffer;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.security.AccessController;
+import java.security.PrivilegedExceptionAction;
+
+final class UnsafeUtils {
```

Comment:
More java doc explaining class methods etc. Richards comments are a great example, he always documents thoroughly and any edge cases.

Reply:
Added comments.